tls/s2n_server_key_exchange.c

```diff
@@ -74,25 +74,24 @@ int s2n_server_key_recv(struct s2n_connection *conn)
 
     /* Parse the KEX data into whatever form needed and save it to the connection object */
     GUARD(s2n_kex_server_key_recv_parse_data(key_exchange, conn, &kex_data));
-
     return 0;
 }
 
-int s2n_ecdhe_server_key_recv_read_data(struct s2n_connection *conn, struct s2n_blob *data_to_verify, union s2n_kex_raw_server_data *raw_server_data)
+int s2n_ecdhe_server_key_recv_read_data(struct s2n_connection *conn, struct s2n_blob *data_to_verify, struct s2n_kex_raw_server_data *raw_server_data, int num_d)
```

Comment:
This implementation doesn't match the forward declaration. Is this pull request a self consistent set of changes?

Reply:
Not sure where the int num_d came from, that is a typo or copy/past or refactor issue. I've removed that and am checking for other such issues.